core/stores/sqlx/utils.go

```diff
@@ -67,6 +67,8 @@ func format(query string, args ...interface{}) (string, error) {
 
 			writeValue(&b, args[argIndex])
 			argIndex++
+		case ':':
```

Comment:
`case ':', '$':`

I think it's better.

Reply:
ok